File: wEBcMD/Impl/DeleteAdress.cs
using System;
using System.Reflection;
using System.Collections.Generic;

namespace wEBcMD
{
   public partial class DeleteAdressWrapper : CommandWrapper
   {
      /// <summary>
      /// Delete the Adress with the given id.
      /// Returns the Adress which was deleted.
      /// </summary>

		public partial AdressDTO DeleteAdress(Guid id)
		{
			Log.Trace($"Implementation in {MethodBase.GetCurrentMethod()}");
			return default;
		}
   };

}

File: wEBcMD/ClientApp/src/impl/DeleteAdress.ts
import { Guid} from "guid-typescript";
import { DeleteAdressBase } from "../api/DeleteAdressBase";
import { CommandDTO } from "../api/CommandDTO";

/**
 * Delete the Adress with the given id.
 *          Returns the Adress which was deleted.
 */
export class DeleteAdress extends DeleteAdressBase {

   constructor(dto?: CommandDTO, type?: Guid){super(dto, type ? type : DeleteAdressBase.TypeId)}

};